src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -21,6 +21,11 @@
 
 		<!-- Required for C# Hot Reload -->
 		<UseInterpreter Condition="'$(Configuration)' == 'Debug'">True</UseInterpreter>
+
+		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
```

Comment:
I'm not sure if the bot fixes space vs. tab in template contents, does it? If not, would be good to update these template files in the PR so customers don't end up with mixed whitespace.

Reply:
Sure fixed to tabs